docs/source/outline.rst

```diff
@@ -7,7 +7,7 @@ Project Outline
 Current plan for the StreamWebs rewrite. For a visual representation, 
 refer to `this`_.
 
-For now, the site's homepage will be a static intro that contains a brief
+For now, the site's homepage will be a static intro that contains a brief 
```

Comment:
@athai wrote the outline. I think the "For now" is there is because it might not be static in the future?